src/Components/Results/Programs/ProgramCard.tsx

```diff
@@ -9,10 +10,26 @@ type ProgramCardProps = {
 const ProgramCard = ({ program }: ProgramCardProps) => {
   const { uuid } = useParams();
 
+  const estimatedAppTime = program.estimated_application_time;
+  const estimatedSavings = program.estimated_value;
+
   return (
-    <div>
-      <ResultsTranslate translation={program.name} />
-      <Link to={`/${uuid}/results/benefits/${program.program_id}`}>Apply</Link>
+    <div className="result-program-container">
+      <div className="result-program-more-info">
+        <ResultsTranslate translation={program.name} />
```

Comment:
You assign `estimatedAppTime` and `estimatedSavings` to variables, but you inline `program.name`. Could you either move `program.name` to it's own variable, or inline the other variables.

Reply:
Updated. Thanks!